packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -771,10 +771,10 @@ const StartVisitForm: React.FC<StartVisitFormProps> = ({
               <div className={styles.sectionTitle}></div>
               <div className={styles.sectionField}>
                 <VisitFormExtensionSlot
-                  name="visit-form-bottom-slot"
+                  name="visit-form-top-slot"
```

Comment:
Basically same feedback as before... there is already a top slot, at the top of the component. This is a different slot, at the bottom of the component.